src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,25 @@
+import styles from "./Checkbox.module.scss";
+
+type CheckboxProps = {
+  checked: boolean;
+  onChange: () => void;
+  className?: string;
+};
+
+const Checkbox = ({ checked, onChange, className }: CheckboxProps) => {
+  return (
+    <div className={styles["checkbox-wrapper"]}>
```

Comment:
checkboxWrapper

Reply:
the same here, classname CheckboxWrapper ?

Reply:
Can be, but personally I would use something like Component or ComponentContainer - Wrapper is something what wrapps elements on lower level IMHO. Just remember to pick one and stick to it

Reply:
like above